src/mlpack/methods/amf/termination_policies/simple_residue_termination.hpp

```diff
@@ -41,7 +41,15 @@ class SimpleResidueTermination
    */
   SimpleResidueTermination(const double minResidue = 1e-5,
                            const size_t maxIterations = 10000)
-      : minResidue(minResidue), maxIterations(maxIterations) { }
+      : minResidue(minResidue), 
```

Comment:
GitHub won't let me suggest it, but I think the 'more correct' way to style this according to our style guide is to put the `:` on the previous line (e.g. `= 10000) :`) and then move all these initialization members over two spaces.

Reply:
sure! Will change that!